jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/HttpChannelState.java

```diff
@@ -700,7 +700,7 @@ public void run()
                             catch (Throwable x)
                             {
                                 if (LOG.isDebugEnabled())
-                                    LOG.debug("{} while invoking onTimeout listener {}", x.toString(), listener, x);
+                                    LOG.atDebug().setCause(x).log("{} while invoking onTimeout listener {}", x.toString(), listener);
                                 else
                                     LOG.warn("{} while invoking onTimeout listener {}", x.toString(), listener);
```

Comment:
You should not change the level you log at based on what level is enabled on the logger. The `isXxxxEnabled` methods should generally only be used for performance optimisation, avoiding building objects to pass them to a logger that will then discard them.